CHANGELOG.md

```diff
@@ -121,7 +124,7 @@ Here's the annotation we added to `Effect`:
 type role Effect representational
 ```
 
-Conversely, we might want to strengthen the roles of parameters with invariants invisible to the type system. Maps are the canonical example of this: the shape of their underlying tree rely on the `Ord` instance of their keys, but the `Ord` instance of a newtype may behave differently than the one of the wrapped type so it would be unsafe to allow coercions between `Map k1 a` and `Map k2 a`, even when `Coercible k1 k2` holds.
+Conversely, we might want to strengthen the roles of parameters with invariants invisible to the type system. Maps are the canonical example of this: the shape of their underlying tree rely on the `Ord` instance of their keys, but the `Ord` instance of a newtype may behave differently than the one of the wrapped type so it would be unsafe to allow coercions between `Map k1 a` and `Map k2 a` when `Coercible k1 k2` holds.
```

Comment:
I actually think this was clearer before. Now, it reads like it _would_ be safe to allow coercions between `Map k1 a` and `Map k2` a as long as there is no `Coercible k1 k2` instance.

Reply:
Let’s rollback this change then.